website/docs/r/subnet.html.markdown

```diff
@@ -8,30 +8,20 @@ description: |-
 
 # ibm\_subnet
 
- This resource provides portable and static subnets that consist of either IPv4 and IPv6 addresses. Users are able to create 
-public portable subnets, private portable subnets, and public static subnets with an IPv4 option and public portable subnets and public static 
-subnets with an IPv6 option. 
+This resource provides portable and static subnets that consist of either IPv4 and IPv6 addresses. Users are able to create 
+public portable subnets, private portable subnets, and public static subnets with an IPv4 option, and public portable subnets and public static subnets with an IPv6 option. 
  
 The portable IPv4 subnet is created as a seconday subnet on a VLAN. IP addresses in the portable subnet can be assigned as secondary IP 
- addresses for IBM resources in the VLAN. Each portable subnet has a default gateway IP address, network IP address, and broadcast 
- IP address. For example, if a portable subnet is `10.0.0.0/30`, `10.0.0.0` is a network IP address, `10.0.0.1` is a default gateway IP address, 
- and `10.0.0.3` is a broadcast IP address. Therefore, only `10.0.0.2` can be assigned to IBM resources as a secondary IP address. 
- Number of usuable IP addresses is `capacity` - 3. If `capacity` is 4, the number of usuable IP addresses is 4 - 3 = 1. If `capacity` is 8, the 
- number of usuable IP addresses is 8 - 3 = 5. For additional details, refer to [Static and Portable IP blocks](https://knowledgelayer.softlayer.com/articles/static-and-portable-ip-blocks).
+addresses for IBM resources in the VLAN. Because each portable subnet has a default gateway IP address, network IP address, and broadcast IP address, the number of usuable IP addresses is `capacity` - 3. A `capacity` of 4 means that the number of usuable IP addresses is 1; a `capacity` of 8 means that the number of usuable IP addresses is 5. For example, consider a portable subnet of `10.0.0.0/30` that has `10.0.0.1` as a default gateway IP address, `10.0.0.0` as a network IP address, and `10.0.0.3` as a broadcast IP address. Only `10.0.0.2` can be assigned to IBM resources as a secondary IP address. For additional details, refer to [Static and Portable IP blocks](https://knowledgelayer.softlayer.com/articles/static-and-portable-ip-blocks).
```

Comment:
Should `usuable IP addresses` be `usable IP addresses`?

Reply:
yes, nice catch.